index.js

```diff
@@ -41,6 +41,10 @@ function HyperDB (storage, key, opts) {
 
   this.key = typeof key === 'string' ? Buffer.from(key, 'hex') : key
   this.discoveryKey = this.key ? hypercore.discoveryKey(this.key) : null
+  this.sourceFeedOpts = {
+    secretKey: opts.secretKey || null,
+    storeSecretKey: opts.hasOwnProperty('storeSecretKey') ? opts.storeSecretKey : true
```

Comment:
can we make this default false? so it's consistent across hypercore and hyperdb?

Reply:
As I read it in Hypercore it's true if unset and false only if set to false:
https://github.com/mafintosh/hypercore/blob/master/index.js#L69
That's why I defaulted to true unless set explicitly - or did I get it wrong?